src/khoj/interface/web/chat.html

```diff
@@ -563,6 +564,148 @@
             document.getElementById("chat-body").scrollTop = document.getElementById("chat-body").scrollHeight;
         }
 
+        function openFileBrowser() {
+            event.preventDefault();
+            var overlayText = document.getElementById("dropzone-overlay");
+            var dropzone = document.getElementById('chat-body');
+
+            if (overlayText == null) {
+                dropzone.classList.add('dragover');
+                var overlayText = document.createElement("div");
+                overlayText.innerHTML = "Select a file to share it with Khoj";
```

Comment:
Can we add a hint that you can also just drag and drop files onto the chat interface to upload them for indexing?

Reply:
I suspect this text won't really be visible to most people once the file picker pops up, but sure, doesn't hurt.